pyproject.toml

```diff
@@ -9,6 +9,7 @@ python = "^3.11"
 openai = "^1.11.1"
 discord = "^2.1.0"
 urllib3 = "<2.0"
+seaborn = "0.13.2"
```

Comment:
I'm guessing this is for Bryce's branch?

Reply:
Yes!

Reply:
I will remove it for now and when Bryce merges we can add that in.